openslides/motions/static/templates/motions/motion-detail.html

```diff
@@ -36,24 +35,188 @@
+              </button>
+            </div>
+          </form>
+          <div ng-show="!poll.isEditMode && poll.yes">
+            <!-- yes -->
+            <div class="result_label">
+              <i class="fa fa-thumbs-up"></i>
+              <translate>Yes</translate>:
+            </div>
+            <div><uib-progressbar value="poll.getYesPercent(true)" type="success"></uib-progressbar></div>
+            <div class="result_value">{{ poll.yes }} {{ poll.getYesPercent() }}</div>
+            <!-- no -->
+            <div class="result_label">
+              <i class="fa fa-thumbs-down"></i>
+              <translate>No</translate>:
+            </div>
+            <div><uib-progressbar value="poll.getNoPercent(true)" type="danger"></uib-progressbar></div>
+            <div class="result_value">{{ poll.no }} {{ poll.getNoPercent() }}</div>
+            <!-- abstain -->
+            <div class="result_label">
+              <b>&empty;</b>
+              <translate>Abstain</translate>:
+            </div>
+            <div><uib-progressbar value=" poll.getAbstainPercent(true)" type="warning"></uib-progressbar></div>
+            <div class="result_value">{{ poll.abstain }} {{ poll.getAbstainPercent() }}</div>
+            <hr class="smallhr" ng-if="poll.votesvalid || poll.votesinvalid">
+            <!-- valid votes -->
+            <div ng-if="poll.votesvalid">
+              <div class="result_label">
+                <i class="fa fa-check"></i>
+                <translate>Valid votes</translate>:
+              </div>
+              <div class="result_value">{{ poll.votesvalid }} {{ poll.getVotesValidPercent() }}</div>
+            </div>
+            <!-- invalid votes -->
+            <div ng-if="poll.votesinvalid">
+              <div class="result_label">
+                <i class="fa fa-ban"></i>
+                <translate>Invalid votes</translate>:
+              </div>
+              <div class="result_value">{{ poll.votesinvalid }} {{ poll.getVotesInvalidPercent() }}</div>
+            </div>
+            <hr class="smallhr" ng-if="poll.votescast">
+            <!-- votes cast -->
+            <div ng-if="poll.votescast">
+              <div class="result_label">
+                <b>&sum;</b>
+                <translate>Votes cast</translate>:
+              </div>
+              <div class="result_value">{{ poll.votescast }} {{ poll.getVotesCastPercent() }}</div>
+            </div>
+          </div>
+      </ol>
+      <button ng-if="motion.allowed_actions.create_poll" ng-click="create_poll()" class="btn btn-default btn-sm">
+        <i class="fa fa-bar-chart fa-lg"></i>
+        <translate>New poll</translate>
+      </button>
+
       <h3 translate>Category</h3>
       {{ motion.category.name }}</a>
 
-      <h3 translate>Voting result</h3>
-      -
+      <h3 translate>Tags</h3>
+      <span ng-repeat="tag in motion.tags">
+        <span class="label label-default">
+          {{ tag.name }}
+        </span>
+        &nbsp;
```

Comment:
Is this line really necessary? Can it be fixed by a css-rule for the span above? (margin-right:5px)